### ISSUE TITLE
Standups: Design recurrence MVP

### ISSUE THREAD

**Issue by enriquesanchez** [ParabolInc/parabol#6747]
This issue is related to #5798, but I want to keep both as separate because on this one we will be exploring at the most bare MVP we can implement so we can start having a very basic version of recurrence for standups.

On this issue in particular, **we won't be looking at more complex settings such as timezones, start and end times, or more advanced scheduling options.**

## Acceptance Criteria 
- Users can set a very basic version of recurrence for their standups

### Estimated effort:
- 3 hrs for design exploration

**Comment by enriquesanchez** [comment]
@jmtaber129 @lorena-cure @BartoszJarocki 

Here's an initial suggestion for how to approach a super basic MVP of recurrence. As we discussed earlier, we want the most bare bones setting we can come up with some we can remove the friction of starting/stopping a standup meeting, without getting too caught up with complex details such as timezones, times, etc.

The idea is to have a single checkbox:

![imagen](https://user-images.githubusercontent.com/3276087/174402463-01874fb8-5657-4095-9440-b4eadc899ea7.png)

And if I turn it on:

![imagen](https://user-images.githubusercontent.com/3276087/174402500-b83a24bc-8b86-4ed9-8ac9-246918e330d9.png)


**Suggestions on improving the copy to make everything more clear are welcome** 🙏 

How do y'all expect the open and close times to work? We had discussed using the timezone of whoever creates the meeting as default, and then set it to open at 12:00am and close at 11:59pm.

Does that still sound reasonable to you?

**Comment by BartoszJarocki** [comment]
@enriquesanchez thank you for working on this! 

Here are a few things missing 
1. We need a way to stop the recurrence
2. If we're closing the meeting automatically, we need to show how much time is left before closing the meeting to avoid unnecessary anxiety
3. We'll need a reminder to do provide the response on time

**Comment by jmtaber129** [comment]
This looks good!

I agree with @BartoszJarocki that we need each of those things.  I think we can end recurrence in the option menu where you can currently end the meeting.  

> when the meeting closes, the information will be lost, right?

I think this depends on the data model used to represent a recurring meeting, but the information should still be in a timeline summary card where you can access the closed meeting (similar to current standups).


I think if we have the time zone of the current user, use 12:01a + 11:59p of that time zone as the start + end times.  If we don't have it, default to a north American time zone.  We (devs) should, however, make sure that any data models we create can support other times.  FWIW my weak preference would be to support on the backend any first-pass recurrence options - days of the week, start time, and end time - to avoid additional otherwise-unnecessary migrations when we add those options to the frontend.

**Comment by enriquesanchez** [comment]
Thanks for the feedback @lorena-cure @BartoszJarocki @jmtaber129. I've addressed your comments below:

> We need a way to stop the recurrence

![image](https://user-images.githubusercontent.com/3276087/177635367-1b023e2d-3111-4b83-b397-67f80800b2fc.png)

As James suggested, recurrence can be stopped from the options menu on the top right corner. I think we should also allow for recurrence to be set from that menu. The use case is I started the activity without recurrence and later I changed my mind, or I stopped recurrence from an ongoing standup and then I changed my mind and want to set it back:

![image](https://user-images.githubusercontent.com/3276087/177635603-db077886-4b06-40ff-ba7b-bd832180fe3d.png)

>we need to show how much time is left before closing the meeting

![image](https://user-images.githubusercontent.com/3276087/177636239-a303b13d-d0ae-42b4-a3ab-916bd05db22d.png)

We can add a little time left indicator at the top-center.

>We'll need a reminder to do provide the response on time

I think this should be handled via Slack? Similar to how we send a notification when a time or deadline has been set on a meeting, we can do the same for the reminder. Let's just start by choosing an arbitrary time to send the notification, something like 8 hrs before it ends? @BartoszJarocki
